diffusers/image_2_image_using_diffusers.ipynb

```diff
@@ -8048,8 +8048,7 @@
     {
```

Comment:
<p>Use <code>image</code> instead of <code>init_image</code>  (deprecated)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>